backend/src/openarchiefbeheer/logging/templates/logging/destruction_list_finalized.txt

```diff
@@ -0,0 +1,4 @@
+{% load i18n %}
+{% blocktranslate trimmed with timestamp=log.timestamp|date:"c" list_name=log.content_object.name archivaris=log.extra_data.archivaris.username %}
+   Destruction list "{{ list_name }}" was finalized by archivist {{ archivaris }}.
+{% endblocktranslate %}
```

Comment:
I think that maybe there is confusion about the issue: 

The title of the issue is: "Geen opmerkingenveld voor doorsturen aan Archivaris"

So if I understand this correctly, this is in the step where all "normal" reviewers have approved and the list is in status "internally_reviewed". The record manager then needs to mark the list as "final" which changes it's status to "ready_for_archivist" and assigns the archivist to it. 

From this log, it looks like the archivist is making the list final, while it is the record manager.